potodo/_github.py

```diff
@@ -1,32 +1,45 @@
 import requests
 
-from subprocess import getoutput
+from subprocess import check_output
 
 
-def get_repo_url():
-    url = getoutput("git remote get-url --all upstream")
+def get_repo_url() -> str:
+    """
+    Tries to get the repository url from git commands
+    """
+    url: str = str(check_output("git remote get-url --all upstream"))
     if "fatal" in url:
-        url = getoutput("git remote get-url --all origin")
+        url = str(check_output("git remote get-url --all origin"))
```

Comment:
A better way to get a string from check_output is to pass `text=True` (or `universal_newlines=True` for backward compatibility).

giving it to str will just repr the bytes:
```
>>> str("é".encode())
"b'\\xc3\\xa9'"
```